Services/CfCli/CfCliService.cs

```diff
@@ -44,7 +44,7 @@ public class CfCliService : ICfCliService
         public static string V6_StartAppCmd = "start";
         public static string V6_DeleteAppCmd = "delete -f"; // -f avoids confirmation prompt
         internal static string V6_GetOrgsRequestPath = "GET /v2/organizations";
-        internal static string V6_GetSpacesRequestPath = "GET /v2/spaces"; 
+        internal static string V6_GetSpacesRequestPath = "GET /v2/spaces";
```

Comment:
Yeah these constant strings are a bit outdated now; since we're using the CfApiClient to get orgs/spaces/apps, much of the code in `CfCliService` isn't being used at the moment. The "V6" naming is a bit misleading; V7 of the CF CLI still uses these routes (re-validated this just now for `cf orgs`), but yes these would have to be changed to `/v3/...` if we wanted to use the CF CLI and CC API v3.

@zzori-theoriginal and I were chatting recently about what to do with these currently-unused methods in CfCliService - we feel like they're good to keep around so that it's easy to switch between implementations for our "CF Communicator" (i.e. either CfApiClient or CfCliService (or something else!)), but -- that said -- it might be good to rename these variables so it doesn't seem like V7 would have different values.

Reply:
Marking as resolved for now, but happy to rename these vars in a separate refactor (since this PR is specific to displaying recent app logs)